src/app/component/list-details/list-details.component.html

```diff
@@ -109,11 +109,19 @@
                         <md-icon>clear</md-icon>
                     </button>
                     <div class="spacer"></div>
-                    <button md-raised-button
-                            [disabled]="(getUser() | async)?.isAnonymous"
-                            (click)="adaptFilters()"
-                            color="accent">Adapt to character
-                    </button>
+                    <div class="btn-group">
+                        <button md-raised-button
+                                [disabled]="(getUser() | async)?.isAnonymous"
+                                (click)="adaptFilters()"
+                                color="accent">Adapt to character
+                        </button>
+
+                        <button md-raised-button
+                                [disabled]="(getUser() | async)?.isAnonymous"
+                                (click)="resetFilters()"
+                                color="accent">Reset Filters
```

Comment:
I think that you should add translation here, and add it to the "Adapt to character" button to at the same time ;)